prime-dotnet-webapi/Models/Api/EmailPair.cs

```diff
@@ -0,0 +1,8 @@
+namespace Prime.Models.Api
+{
+    public class EmailPair
```

Comment:
More descriptive name please?

Reply:
right, the class name and the file name are updated.